docs/self-managed/update/administrators/plan-update.md

```diff
@@ -0,0 +1,237 @@
+---
+id: plan-admin-update
+title: "Plan an update"
+description: "Plan your update to Camunda 8.8 on Self-Managed - Administrator guide."
+---
+
+A successful Camunda 8.8 update requires thorough planning. This guide helps you assess your infrastructure, understand operational requirements, and choose the right update strategy for your environment.
+
+## Step 1: Identify your upgrade path
+
+Determine your current and target versions to plan the update path.
+
+### Current version assessment
+
+**First, identify your current Camunda versions:**
+
+```bash
+# Check Kubernetes deployment versions
+kubectl get pods -l app.kubernetes.io/instance=camunda-platform -o jsonpath='{range .items[*]}{.metadata.name}{"\t"}{.spec.containers[0].image}{"\n"}{end}'
+
+# Check Helm chart version
+helm list | grep camunda-platform
+```
+
+### Sequential update requirement
+
+**Important**: Update through each minor version. Do not skip versions.
+
+**Update path examples:**
+
+- **Correct**: 8.5 → 8.6 → 8.7 → 8.8
+- **Incorrect**: 8.5 → 8.8 (skips versions)
+
+**Why?** Each version may include data migrations that later versions require.
+
+### Component coordination
+
+**Plan to update all core components together:**
+
+- Zeebe cluster
+- Operate, Tasklist, Optimize
+- Identity (Keycloak)
+- Elasticsearch/OpenSearch
+
+**Tip**: Use the latest patch release (8.8.x) to get all bug fixes.
```

Comment:
Actually, we document [here](https://docs.camunda.io/docs/next/self-managed/operational-guides/update-guide/introduction/#update-procedure) that this is a "must"